src/Servers/IIS/IntegrationTesting.IIS/src/IISDeployer.cs

```diff
@@ -305,15 +307,32 @@ private static Site FindSite(ServerManager serverManager, string contentRoot)
 
     private void AddTemporaryAppHostConfig(string contentRoot, int port)
     {
-        _configPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("D"));
-        _applicationHostConfig = Path.Combine(_configPath, "applicationHost.config");
-        Directory.CreateDirectory(_configPath);
-        var config = XDocument.Parse(DeploymentParameters.ServerConfigTemplateContent ?? File.ReadAllText("IIS.config"));
+        var multiSite = true;
```

Comment:
```suggestion
        var multiSite = _applicationHostConfig is not null;
```

Is this clearer? It seems like `multisite` is always true if we have a preexisting host config?